// slack/methods/conversations.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _conversations = require('./conversations.archive');

var _conversations2 = _interopRequireDefault(_conversations);

var _conversations3 = require('./conversations.close');

var _conversations4 = _interopRequireDefault(_conversations3);

var _conversations5 = require('./conversations.create');

var _conversations6 = _interopRequireDefault(_conversations5);

var _conversations7 = require('./conversations.history');

var _conversations8 = _interopRequireDefault(_conversations7);

var _conversations9 = require('./conversations.info');

var _conversations10 = _interopRequireDefault(_conversations9);

var _conversations11 = require('./conversations.invite');

var _conversations12 = _interopRequireDefault(_conversations11);

var _conversations13 = require('./conversations.join');

var _conversations14 = _interopRequireDefault(_conversations13);

var _conversations15 = require('./conversations.kick');

var _conversations16 = _interopRequireDefault(_conversations15);

var _conversations17 = require('./conversations.leave');

var _conversations18 = _interopRequireDefault(_conversations17);

var _conversations19 = require('./conversations.list');

var _conversations20 = _interopRequireDefault(_conversations19);

var _conversations21 = require('./conversations.members');

var _conversations22 = _interopRequireDefault(_conversations21);

var _conversations23 = require('./conversations.open');

var _conversations24 = _interopRequireDefault(_conversations23);

var _conversations25 = require('./conversations.rename');

var _conversations26 = _interopRequireDefault(_conversations25);

var _conversations27 = require('./conversations.replies');

var _conversations28 = _interopRequireDefault(_conversations27);

var _conversations29 = require('./conversations.setPurpose');

var _conversations30 = _interopRequireDefault(_conversations29);

var _conversations31 = require('./conversations.setTopic');

var _conversations32 = _interopRequireDefault(_conversations31);

var _conversations33 = require('./conversations.unarchive');

var _conversations34 = _interopRequireDefault(_conversations33);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
  archive: _conversations2.default,
  close: _conversations4.default,
  create: _conversations6.default,
  history: _conversations8.default,
  info: _conversations10.default,
  invite: _conversations12.default,
  join: _conversations14.default,
  kick: _conversations16.default,
  leave: _conversations18.default,
  list: _conversations20.default,
  members: _conversations22.default,
  open: _conversations24.default,
  rename: _conversations26.default,
  replies: _conversations28.default,
  setPurpose: _conversations30.default,
  setTopic: _conversations32.default,
  unarchive: _conversations34.default
};
module.exports = exports['default'];
