// slack/methods/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _api = require('./api.test');

var _api2 = _interopRequireDefault(_api);

var _api3 = require('./api.client');

var _api4 = _interopRequireDefault(_api3);

var _bots = require('./bots');

var _bots2 = _interopRequireDefault(_bots);

var _auth = require('./auth.test');

var _auth2 = _interopRequireDefault(_auth);

var _auth3 = require('./auth.revoke');

var _auth4 = _interopRequireDefault(_auth3);

var _channels = require('./channels');

var _channels2 = _interopRequireDefault(_channels);

var _chat = require('./chat');

var _chat2 = _interopRequireDefault(_chat);

var _conversations = require('./conversations');

var _conversations2 = _interopRequireDefault(_conversations);

var _dnd = require('./dnd');

var _dnd2 = _interopRequireDefault(_dnd);

var _emoji = require('./emoji.list');

var _emoji2 = _interopRequireDefault(_emoji);

var _files = require('./files');

var _files2 = _interopRequireDefault(_files);

var _groups = require('./groups');

var _groups2 = _interopRequireDefault(_groups);

var _im = require('./im');

var _im2 = _interopRequireDefault(_im);

var _appsPermissions = require('./apps.permissions.info');

var _appsPermissions2 = _interopRequireDefault(_appsPermissions);

var _appsPermissions3 = require('./apps.permissions.request');

var _appsPermissions4 = _interopRequireDefault(_appsPermissions3);

var _mpim = require('./mpim');

var _mpim2 = _interopRequireDefault(_mpim);

var _oauth = require('./oauth.access');

var _oauth2 = _interopRequireDefault(_oauth);

var _oauth3 = require('./oauth.token');

var _oauth4 = _interopRequireDefault(_oauth3);

var _reactions = require('./reactions');

var _reactions2 = _interopRequireDefault(_reactions);

var _reminders = require('./reminders');

var _reminders2 = _interopRequireDefault(_reminders);

var _pins = require('./pins');

var _pins2 = _interopRequireDefault(_pins);

var _rtm = require('./rtm.client');

var _rtm2 = _interopRequireDefault(_rtm);

var _rtm3 = require('./rtm.start');

var _rtm4 = _interopRequireDefault(_rtm3);

var _rtm5 = require('./rtm.connect');

var _rtm6 = _interopRequireDefault(_rtm5);

var _search = require('./search');

var _search2 = _interopRequireDefault(_search);

var _stars = require('./stars');

var _stars2 = _interopRequireDefault(_stars);

var _team = require('./team');

var _team2 = _interopRequireDefault(_team);

var _usergroups = require('./usergroups');

var _usergroups2 = _interopRequireDefault(_usergroups);

var _users = require('./users');

var _users2 = _interopRequireDefault(_users);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
  apps: {
    permissions: { info: _appsPermissions2.default, request: _appsPermissions4.default }
  },
  api: { test: _api2.default, client: _api4.default },
  auth: { test: _auth2.default, revoke: _auth4.default },
  bots: _bots2.default,
  channels: _channels2.default,
  chat: _chat2.default,
  conversations: _conversations2.default,
  dnd: _dnd2.default,
  emoji: { list: _emoji2.default },
  files: _files2.default,
  groups: _groups2.default,
  im: _im2.default,
  mpim: _mpim2.default,
  oauth: { access: _oauth2.default, token: _oauth4.default },
  reactions: _reactions2.default,
  reminders: _reminders2.default,
  pins: _pins2.default,
  rtm: { client: _rtm2.default, start: _rtm4.default, connect: _rtm6.default },
  search: _search2.default,
  stars: _stars2.default,
  team: _team2.default,
  usergroups: _usergroups2.default,
  users: _users2.default
};
module.exports = exports['default'];
